src/data/strategies/steady-eth.ts

```diff
@@ -0,0 +1,134 @@
+    title: "Patache",
+    href: "https://www.algoreturns.com/patache/",
+    tooltip:
+      "A Strategy Provider is responsible for providing the instructions for a cellar to execute",
+  },
+  strategyBreakdown: {
+    goals: `Catch trends early by predicting the significant movement of prices consistently and accurately. Capture the majority of ETH price breakouts, and limit losses through trailing stops.`,
+
+    highlights: `The cellar:
+
+      - "Risk first" approach - capital preservation is more important than capital growth.
+
+      - Always defined risk for every position prevailing from trade inception until trade exit.
+
+      - Each trade strategy comprises two independent trade orders: 1 "Workhorse" with a fixed target and stop + 1 "Racehorse" with a trailing stop.`,
+    description: `Patache has pursued a pragmatic approach to developing a trading strategy instead of a strict theoretical framework. A foundation of our pragmatic approach is a "risk first" paradigm – capital preservation is more important than capital growth. The strategy emphasizes principal protection and steady, consistent returns while pursuing occasional "home runs."
+
+      The trade management technique of ETH Breakout strategy comprises two components: a workhorse and a racehorse. The point of the workhorse is to nullify risk and capture a small profit/cover transaction cost, and the point of the racehorse is to pursue a larger payoff opportunity. Check the "How it works" section for detailed strategy performance explanations.
+
+      The Strategy is expected to capture most of any positive price breakouts and limit losses through its trailing stops. Due to the nature of the strategy, it is designed to be held over a medium to the long term (6 months to a year). In this time, the benefits of being in the strategy are expected to emerge.`,
+    backtesting: `
+      <div style="display:flex;flex-direction:row;gap:5rem;">
+        <div style="width:50%">
+          Beginning Cellar Value: 500,000
+          Period: May 2021 - September 2022
+          No. of Trades: 72
+          Worst Loss (Single Trade): -6,838
+          Worst drawdown:  7.55%
+          Annualized Sharpe Ratio: 267%
+          Annualized Std.Dev of Return: 10.4%
+        </div>
+        <div style="width:50%">
+          Annualized Mean Return: 27.8%
+          Cumulative profit: 46.6%
+          Win rate: 55%
+          Loss rate: 45%
+          Best month: 8%
+          Worst month: -2.5%
+        </div>
+      </div>
+      Presented results are based on historical backtests. Past performance is not indicative of future results. Actual performance will depend on market conditions.`,
+  },
+  // overrideApy: {
+  //   title: "Backtested APY",
+  //   tooltip:
+  //     "Backtested APY results are based on historical backtests. Past performance is not indicative of future results. Actual performance will depend on market conditions",
+  //   value: "84.15%",
+  // },
```

Comment:
remove if not used

Reply:
I comment this out for the strategy details later